corehq/apps/domain/deletion.py

```diff
@@ -137,9 +138,18 @@ def _delete_web_user_membership(domain_name):
     for web_user in list(active_web_users) + list(inactive_web_users):
         web_user.delete_domain_membership(domain_name)
         if settings.UNIT_TESTING and not web_user.domain_memberships:
+            # log before delete, if user is being deleted
+            _log_web_user_membership_removed(web_user, __name__ + "._delete_web_user_membership")
```

Comment:
is it necessary to have this here? This is for unit tests only and will result in logs getting created every time a domain is deleted that has web users (since mostly they won't have more than one membership)

Reply:
Hey @snopoke 

I see your point. That is one way to test if the logs are getting added correctly.
But I should be able to give user two domain memberships in the test, that way it would hit the `else` block.